tools/update-rust.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -lt 1 ]]; then
+    echo "Usage: $0 $RUST_VERSION"
+    echo
+    echo "Example: $0 1.71.0"
+    exit 2
+fi
+
+TOOLSDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOTDIR=$(realpath "${TOOLSDIR}/..")
+
+VERSION="${1}"
+OUTPUT="${ROOTDIR}/hashes/rust"
+PACKAGE_ROOT="https://static.rust-lang.org"
+METADATA_FILE="metadata.json"
+ARCHES=("x86_64" "aarch64")
+PACKAGES=("rust-std" "rustc" "cargo")
+
+function get_package_url {
+    local package="$1"
+    local arch="$2"
+
+    path=$(grep -e "${package}.*${arch}-unknown-linux-gnu.tar.xz" "${METADATA_FILE}" | cut -d '"' -f 3)
+    path=$(echo "${path//\\}")
+
+    echo "${path}"
+}
+
+function add_package {
```

Comment:
Can we also check the signatures with the `*.tar.xz.asc` files?

Reply:
Good idea - done!